docs/troubleshooting.rst

```diff
@@ -13,7 +13,7 @@ you may perform the following manual setup steps.
 
 1. Install Runhouse on your local box with:
 
-   .. code::
+   .. code:: ipython3
```

Comment:
I think this can be shell, same as the pip install below

Reply:
oh yup! thanks for catching that